Add space and make GitHub icons work in dark mode on webkit.org/team/

annevk opened the pull request:
#### 63fca278d3958bf09565bbe5d959abde2f7397f0
<pre>
Add space and make GitHub icons work in dark mode on webkit.org/team/
<a href="https://bugs.webkit.org/show_bug.cgi?id=246563">https://bugs.webkit.org/show_bug.cgi?id=246563</a>

Reviewed by Tim Nguyen.

* Websites/webkit.org/wp-content/themes/webkit/team.php:

Canonical link: <a href="https://commits.webkit.org/255577@main">https://commits.webkit.org/255577@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/2e8d26fd15328fef44edf413e3afd8d907630a3a

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/92999 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/2201 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/23574 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/102712 "Built successfully") | 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/2211 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/30529 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/85384 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/98826 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/98665 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/1513 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/79474 "Built successfully") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/35/builds/28433 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/83428 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/3/builds/83153 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/71544 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/36926 "Built successfully") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/17048 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/34737 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/18246 "Passed tests") | | 
| [  ~~🛠 🧪 merge~~](https://ews-build.webkit.org/#/builders/74/builds/3864 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/38610 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/60/builds/40845 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/1782 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/40535 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/37434 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on Websites/webkit.org/wp-content/themes/webkit/team.php:
```diff
@@ -17,7 +17,11 @@
         --expertise-shadow: 0px 5px 10px 0px hsla(0, 0%, 100%, 0.1);
         --hover-text-color: hsl(0, 0%, 100%);
     }
+    img {
```
That seems error-prone for the future, maybe add a class name on the Github icon and target that?

annevk replied:
It seems to be the existing convention in the file, also for `em` and `span`.

nt1m replied:
They do have `li` in front (though I don't find that safe enough either tbh). As it stands, your rule could target anything including images in the navbar or footer. Personally I'd just add a `github-icon` class name from the JS.

annevk committed:
Add space and make GitHub icons work in dark mode on webkit.org/team/
https://bugs.webkit.org/show_bug.cgi?id=246563

Reviewed by Tim Nguyen.

* Websites/webkit.org/wp-content/themes/webkit/team.php:

Canonical link: https://commits.webkit.org/255577@main
